### PR TITLE
Improve textile examples autocomplete.

### PR DESCRIPTION
**Pull request by n1k0**
A very simple enhancement and a small bugfix. 

@magopian I was also thinking of searching into the category name as well, because atm if you search for "polo" it shows no results, while we might want to expose t-shirt examples from the *t-shirt/polo* category as a starting point… what do you think?

### CODE REVIEW

**Comment by magopian** [comment]
Yes indeed it would make sense to be able to search on the "category" (product type) also 👍

**Comment by n1k0** [comment]
@magopian ok I'll implement it in this patch

File: src/Data/Textile/Inputs.elm [hunk 1]
```diff
@@ -1049,7 +1051,7 @@ pullLaineAsie =
 jupePolyesterAsie : Query
 jupePolyesterAsie =
     { jupeCotonAsie
-        | materials = [ { id = Material.Id "pa", share = Split.full, spinning = Nothing, country = Nothing } ]
+        | materials = [ { id = Material.Id "pet", share = Split.full, spinning = Nothing, country = Nothing } ]
```
Comment (magopian) [inline src/Data/Textile/Inputs.elm:1054]: ah, well spotted, was an error on my end 👍

**Review by magopian — APPROVED** [review]
Makes sense to order alphabetically instead of "semantically" (all the asia products first, then all the french products and so on...)

**Comment by n1k0** [comment]
Searching from categories looks more complicated than initially considered, I'm landing this and will file a card so we don't forget about it

**Pull request merged**